script/Ciel/Day 10/10_01_CIEL10_5B.txt

```diff
@@ -1,52 +1,57 @@
 [sha:1dd014b7fdff66374d3ba54085710823dded54c3]{
 -- Page 10025, Offset 28029.
 -- 夕日が茶道室を赤く染める。
-The setting sun bathes the tea ceremony room red.
+The setting sun casts a red glow over the tea ceremony room.
 }
 [sha:6afd3e5b6e25df7c7cf437f29acc5b20b30a571f]{
 -- Page 10025, Offset 28030.
 -- 結局、先輩と放課後まで一緒に過ごしてしまった。
-In the end, I spent the whole afternoon with Senpai.
+I ended up spending the entire afternoon here with Senpai.
 }
 [sha:88e81ac01b5995ff132f01fb27f301b8cd5b1e87]{
 -- Page 356, Offset 28031.
 -- 「では今日からわたしと遠野くんはチームですね。ふたりで事件を解決しましょう。
-"We'll form a team from today on then, Tohno-kun. Let's do our best to settle this case together.
+"Well then, I guess we're a team from now on, Tohno-kun.
+ Let's do our best to bring all this to a close together.
```

Comment:
I just kinda hate 事件 because I feel like you don't really refer to goings-on as 'incidents' or 'cases' in EN unless you're like, the cops. I guess we've already committed though.